AppCenterIdentity/AppCenterIdentity/Internals/Storage/MSKeychainAuthTokenStorage.m

```diff
@@ -75,22 +76,20 @@ - (void)saveAuthToken:(nullable NSString *)authToken withAccountId:(nullable NSS
     // Save new array.
     [self setAuthTokensHistoryState:authTokensHistory];
     if (authToken && accountId) {
-      [MS_USER_DEFAULTS setObject:(NSString *)accountId forKey:kMSIdentityMSALAccountHomeAccountKey];
+      [MS_USER_DEFAULTS setObject:(NSString *) accountId forKey:kMSIdentityMSALAccountHomeAccountKey];
     } else {
       [MS_USER_DEFAULTS removeObjectForKey:kMSIdentityMSALAccountHomeAccountKey];
     }
   }
 }
 
-// TODO: Finish the implementation of tokens removal as part of the separate PR.
 - (void)removeAuthToken:(nullable NSString *)__unused authToken {
```

Comment:
Have you compiled the project? `authToken` is declared as unused but it is used.

Reply:
Migrated from other PR afaik; Ill remove directive